banyan/resource_service_k8s.go

```diff
@@ -221,8 +220,11 @@ func expandK8sMetatdataTags(d *schema.ResourceData) (metadatatags service.Tags)
 	descriptionLink := d.Get("description_link").(string)
 	allowUserOverride := d.Get("end_user_override").(bool)
 	banyanProxyMode := "CHAIN"
-	alpInt := d.Get("client_banyanproxy_listen_port").(int)
-	appListenPort := strconv.Itoa(alpInt)
+	alp, ok := d.GetOk("client_banyanproxy_listen_port")
+	appListenPort := ""
+	if ok {
+		appListenPort = strconv.Itoa(alp.(int))
```

Comment:
is there any possibility that this type assertion `alp(int)` could fail and panic?

Reply:
nope @yoshiotu input type is marked integer for that property, so if there is a value it would always be int.